samples/benchmarks/coremark/sample.yaml

```diff
@@ -0,0 +1,20 @@
+sample:
+  name: CoreMark Benchmark
```

Comment:
Looking at other ncs examples I think the yaml file should be stored under nrf/samples/benchmark/coremark